problem_builder/templates/html/mcqblock.html

```diff
@@ -10,10 +10,10 @@ <h4 class="question-title" id="heading_{{ self.html_id }}">{{ self.display_name_
       <label class="choice-label"
              for="{{ self.html_id }}_{{ forloop.counter }}"
              aria-describedby="feedback_{{ self.html_id }} choice_tips_{{ self.html_id }}-{{ forloop.counter }}">
-        <span class="choice-result fa icon-2x"
+        <button class="choice-result fa icon-2x"
```

Comment:
@Agrendalath Changing `span` to `button` has implications in `master`, where not styling is applied on top of the block.
Check [this comment](https://github.com/open-craft/problem-builder/pull/257#discussion_r366371230) for more context.

Reply:
@giovannicimolin  necessary style has already been applied while fixing this for MRQs. So that is its appearing without any UI disturbance here.

Reply:
Thanks for noticing, @giovannicimolin.
@musmanmalik, could you please ping me on this PR after resolving it?

Reply:
@Agrendalath @giovannicimolin  There are no implications of this since necessary style overridden in [for mrqs](https://github.com/open-craft/problem-builder/pull/249/files) and in this PR.
 
Only one scenario left [for](https://github.com/open-craft/problem-builder/pull/257) which i will fix and push in [this pr](https://github.com/open-craft/problem-builder/pull/257) soon.

Reply:
@musmanmalik Note that @giovannicimolin is talking about breaking on the vanilla upstream edx/edx-platform LMS not Apros. Have you tested on the vanilla edx-platform? If not please do and fix the styling issues there as well.

Reply:
@xitij2000 yes I have tested on the vanilla edx-platform as well. It looks like for mrqs and mcqs

![mcqs case](https://user-images.githubusercontent.com/43761905/72432004-dd537b00-37b7-11ea-9d3f-3f340f163d6b.png)
![mrq case](https://user-images.githubusercontent.com/43761905/72432015-e47a8900-37b7-11ea-98b9-9c047818ac6e.png)


Further I have also pushed style changes for [PR](https://github.com/open-craft/problem-builder/pull/257) for assessment case which now looks as following
![assesment case](https://user-images.githubusercontent.com/43761905/72432144-21df1680-37b8-11ea-8eff-3c414dbf1f50.png)

cc: @giovannicimolin

Reply:
The checkmark padding looks off. Can you add back the vertical padding so it aligns properly with the button text?
Also, you can just use `background: none`.

Reply:
@xitij2000 Is that fine now?
![after padding](https://user-images.githubusercontent.com/43761905/72522587-ddb74900-387f-11ea-949e-881f1bd3a646.png)
![after padding1](https://user-images.githubusercontent.com/43761905/72522597-e27bfd00-387f-11ea-87b0-b17ce0f26e14.png)

Reply:
@xitij2000  could you please look into this?

Reply:
@musmanmalik, I'll review this PR on Monday.

Reply:
@Agrendalath Please review this.

Reply:
This one seems to be outdated. Marking as resolved for now.